aws/cloudformation/components/database.yml.erb

```diff
@@ -134,6 +138,11 @@
         # BEGIN: Dynamic configuration settings. Changing these settings does NOT require nor trigger restart of instances.
         slow_query_log: 1
         innodb_monitor_enable: all
+        tx_isolation: READ-COMMITTED
+
+        # Prevent a slow query from degrading instance or cluster performance.
+        # https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_execution_time
+        max_execution_time: 30000
```

Comment:
`tx_isolation` vs `transaction_isolation` below. Is that an intentional difference?